index.html

```diff
@@ -3,34 +3,38 @@
-                            </svg>
```

Comment:
- [ ] As requested by the previous reviewer, you should wrap your logo with `a` so someone can easily navigate your website by clicking on the logo to get back to the home page. :100:

Reply:
@Owusu-Desmond  please the logo is wrapped with an a tag already